### PR TITLE
Add fixed width/height to `PostListing` thumbnail parent

### PR DESCRIPTION
**Pull request by alectrocute**
Hi Lemdevs!

I noticed our thumbnails were looking inconsistent, so made an attempt to keep everything 1:1, at 5rem*5rem.

Before:

Screenshot 2023-06-27 at 5 33 20 PM

After:

Screenshot 2023-06-27 at 5 31 08 PM

Screenshot 2023-06-27 at 5 31 11 PM

Let me know your thoughts and/or if there's a better way to accomplish this. Our thumbnail generating function is a little bit of a mess, could be a good opportunity for refactor here.

Thanks!

### CODE REVIEW

**Review by SleeplessOne1917 — APPROVED** [review]
LGTM

**Comment by jsit** [comment]
I think a col-auto next to a col-9 col-sm-10 or whatever it is might not work always. Hold pleaase

**Comment by jsit** [comment]
Screenshot 2023-06-27 at 6 28 53 PM

**Comment by jsit** [comment]
Even if they're both col-auto, it doesn't work; the content part can wrap. hold please

Screenshot 2023-06-27 at 6 30 48 PM

**Review by jsit — CHANGES_REQUESTED** [review]
needs tweaking

File: src/shared/components/post/post-listing.tsx [hunk 1]
```diff
@@ -1433,7 +1433,7 @@ export class PostListing extends Component<PostListingProps, PostListingState> {
             )}
             <div className="col flex-grow-1">
               <div className="row">
-                <div className="col-sm-3 col-lg-2 pe-0 post-media">
+                <div className="col-auto pe-0 post-media">
                   <div className="">{this.thumbnail()}</div>
                 </div>
                 <div className="col-12 col-sm-9 col-lg-10">
```
Comment (jsit) [inline src/shared/components/post/post-listing.tsx:1439]:
```suggestion
                <div className="col flex-grow-0 post-media">
                  <div className="">{this.thumbnail()}</div>
                </div>
                <div className="col flex-grow-1">
```

**Comment by alectrocute** [comment]
Made further tweaks off your work, @jsit.
Screenshot 2023-06-27 at 8 01 05 PM
Screenshot 2023-06-27 at 8 01 10 PM
Screenshot 2023-06-27 at 8 01 14 PM


Screenshot 2023-06-27 at 8 00 55 PM

**Comment by jsit** [comment]
Do we need a `mb-2` on this?

Screenshot 2023-06-27 at 8 34 34 PM

Also is `text-opacity-100-hover` doing anything on that icon?

**Review by jsit — CHANGES_REQUESTED** [review]
Oops sorry, I forgot to submit this review. Bootstrap already gives us overflow-hidden

File: src/assets/css/main.css [hunk 2]
```diff
@@ -227,6 +227,10 @@ hr {
   text-overflow: ellipsis;
 }
 
+.overflow-hidden {
```
[the remark this answers is not in the document]
Comment (alectrocute) [reply src/assets/css/main.css:230]: Wasn't sure about this. I'll use the BS class instead!

[the remark this answers is not in the document]
Comment (alectrocute) [reply src/assets/css/main.css]: Removed it! You were right – It was duplicative.

**Review by dessalines — APPROVED** [review]

**Comment by SleeplessOne1917** [comment]
@jsit Your review is still marked as need changes. Is the PR good to go now?

**Review by jsit — APPROVED** [review]

**Pull request merged**